src/CommunityToolkit.Maui/Extensions/CryptographyExtensions.shared.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace CommunityToolkit.Maui.Extensions;
+
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+/// <summary>Cryptography extensions.</summary>
+public static class CryptographyExtensions
+{
+	/// <summary>A hash function producing a 128-bit hash value (16 Bytes, 32 Hexadecimal characters)</summary>
+	/// <param name="source">Source string.</param>
+	/// <param name="separator">Separator.</param>
+	/// <returns>Array of 16 bytes.</returns>
+	public static string GetMd5Hash(this string source, string separator = "-")
```

Comment:
maybe keep the separator as a string.Empty because it is parameter you use in our lib.

Reply:
@VladislavAntonyuk I can't use `string. Empty` as this causes "CS1736: Default parameter value for 'separator' must be a compile-time constant."